PyU4V/replication.py

```diff
@@ -226,11 +227,16 @@ def create_storage_group_snapshot(
         :param snap_name: snapshot name -- str
         :param ttl: Time To Live -- str
         :param hours: if TTL is in hours instead of days -- bool
+        :param secure: sets secure snapshot, snapshot created with secure
+        option can not be deleted before ttl expires -- bool
```

Comment:
See comment above

Reply:
Still needs addressed

Reply:
This is still not correct, you should be seeing sphinx build warnings at the very least for things like this I believe, did you try building them yet to make sure its building cleanly?